optuna_dashboard/_importance.py

```diff
@@ -97,7 +98,11 @@ def _get_param_importances(
 
 
 def get_param_importance_from_trials_cache(
-    storage: BaseStorage, study_id: int, objective_id: int, trials: list[FrozenTrial]
+    storage: BaseStorage,
+    study_id: int,
+    objective_id: int,
+    trials: list[FrozenTrial],
+    inmemory_cache: InMemoryCache,
```

Comment:
It would be better to place the argument `inmemory_cache` at the beginning to match the other functions.